src/sentry/mediators/external_requests/util.py

```diff
@@ -52,14 +53,30 @@ def send_and_save_sentry_app_request(url, sentry_app, org_id, event, **kwargs):
 
     buffer = SentryAppWebhookRequestsBuffer(sentry_app)
 
+    slug = sentry_app.slug
```

Comment:
should I factor this out? I wasn't sure which file would be good to put it in.

Reply:
@ceorourke You should put this as a class method of the sentry app so we don't have to repeat this in multiple places